src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -76,15 +81,24 @@ func newBundleCommand(fs afero.Fs) *cobra.Command {
 			logsLimit, err := units.FromHumanSize(logsSizeLimit)
 			out.MaybeDie(err, "unable to parse --logs-size-limit: %v", err)
 
-			err = executeBundle(cmd.Context(), fs, cfg, cl, admin, logsSince, logsUntil, int(logsLimit), timeout)
+			err = executeBundle(cmd.Context(), fs, cfg, cl, admin, logsSince, logsUntil, int(logsLimit), timeout, pathname)
 			out.MaybeDie(err, "unable to create bundle: %v", err)
 		},
 	}
 	command.Flags().StringVar(
 		&adminURL,
-		"admin-url",
+		config.FlagAdminHosts2,
+		"",
+		"Comma-separated list of admin API addresses (<IP>:<port>)",
+	)
+	command.Flags().StringVar(&adminURL, "admin-url", "", "")
+	command.Flags().MarkDeprecated("admin-url", "use --"+config.FlagAdminHosts2)
+
+	command.Flags().StringVar(
+		&outFile,
+		"output",
```

Comment:
I thought we had another command that printed to a file, but can't remember it offhand. Do you know of it / if it exists? I'm wondering because I'm not seeing the `--output` flag defined elsewhere yet.

Reply:
We only have `--out` here https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/iotune/iotune.go#L83

However, I decided to use output here since I think is more commonly used (See `gcc` and `sort` use `--output` to write the result to a file instead of stdout)

I also was thinking of `--output-file` to be consistent with the flag of `zip`. wdyt?

Reply:
I see, I might have been thinking about `rpk cluster config export --filename`

Perhaps we should do another table of which flag is most common across many CLI tools?